CONTRIBUTING.md

```diff
@@ -0,0 +1,173 @@
+
+If you think that you've found a problem in the code, you can report bugs by opening an [issue](https://github.com/lisushka/Winnie_Bot/issues) on GitHub.
+
+## Making improvements
+
+### Code
+
+#### Requirements
+
+You'll need the following dependencies to develop Winnie locally:
+
+* [Docker](https://www.docker.com/)/[Docker Compose](https://docs.docker.com/compose/)
+* [Node.js](https://nodejs.org/en/) 14.0 or higher
+* [Yarn](https://yarnpkg.com/)
+
+#### Running Winnie for local development
+
+<ol>
+  <li>
+    Clone the repo, and open the repo directory:
+    <br>
+    <pre>$ git clone https://github.com/aigeroni/Winnie_Bot.git winnie_bot && cd winnie_bot</pre>
+  </li>
+  <li>
+    Install dependencies with <code>yarn</code>:
+    <br>
+    <pre>$ yarn install</pre>
+  </li>
+  <li>
+    Create <code>.env</code> file:
+    <br>
+    <pre>$ cp .env.sample .env</pre>
+  </li>
+  <li>
+    Setup your environment variables in the <code>.env</code> file:
+  </li>
+  <li>
+    Start the Docker processes:
+    <br>
+    <pre>$ yarn docker:local:up -d</pre>
+  </li>
+  <li>
+    Run the migrations to set up the database:
+    <br>
+    <pre>$ yarn typeorm migration:run</pre>
+  </li>
+  <li>
+    Start Winnie:
+    <br>
+    <pre>$ yarn start</pre>
+  </li>
+  <li>
+    In seperate terminal windows, start the job workers:
+    <br>
+    <pre>$ yarn start:worker:goal</pre>
+  </li>
+</ol>
+
+#### `package.json` scripts
+
+You can use the following scripts on your local CLI to complete tasks:
+
+* `yarn build` - Compile Winnie's TypeScript code into plain JavaScript
+* `yarn docker:local:down` - Shut down local Docker processes
+* `yarn docker:local:up` - Start up local Docker processes
+* `yarn lint:all` - Run ESLint over Winnie's code
+* `yarn lint:fix` - Run ESLint over Winnie's code, fixing basic linting errors
+* `yarn start` - Compile Winnie and start up the bot process
+* `start:worker:goal` - Start the goals worker, needed for goals to complete.
+* `yarn typeorm` - Access the [TypeORM CLI](https://typeorm.io/#/using-cli)
+
+#### Code standards
+
+We use the following standards whilst coding:
+
+* `ts-standard` for linting TypeScript files
+* `terraform fmt` for linting Terraform infrastructure as code
+
+We favour heavily commented code; this makes it easier for all of our contributors to understand how Winnie works.  We may ask you to improve your comments if we think that they need to be clearer.
+
+#### Pull requests
+
+Pull requests should explain what you've done at a high level.  If you're making major changes to the code, then a detailed explanation in both the pull request and the comments is ideal.
+
+We have a pull request template, which you can edit accordingly.  Feel free to delete any headings that aren't relevant.
+
+### Documentation
+
+If you want to update Winnie's documentation, you'll need to submit a pull request in the [docs repo]().  You can find instructions for doing so there.
```

Comment:
you have a repo link every where else in these files, should probably put it here too?